### ISSUE TITLE
Not an issue

### ISSUE THREAD

**Issue by iNoSec2** [Orange-Cyberdefense/GOAD#6]
Bonjour,
Juste pour savoir si le projet était toujours actif et en dev pour une implémentation d'autres vulnérabilités? Il serait intéressant d'ajouter un serveur SQL+WEB et CIFS. En ajoutant les vulnérabilités sur la délégation avec le serveur Web et sql etc etc
Merci pour tout, votre blog ainsi que toutes vos ressources sont vraiment top!!

**Comment by Mayfly277** [comment]
Bonjour,
oui le projet est toujours très actif même. Cependant je n'ai pas encore eu le temps de tout mettre au propre et de faire suffisamment de test pour préparer une release public.

Dans les cartons (tout ce que je voudrais implémenter qui est fait ou en cours) : 
- Ajout serveur IIS avec une page d'upload de fichier asp (prêt)
- Ajout d'un serveur MSSQL avec les droits impersonate sur le user sa pour de l'exécution via xp_cmdshell (prêt)
- Ajout de constrained delegation (prêt)
- Ajout de plusieurs domaines pour tester des techniques de forest to forest (en cours)
- Ajout d'un autre serveur MSSQL en trusted link (todo)
- Ajout d'adcs et de template vulnérables pour tester les attaques esc1-8 (todo)

Sinon j'ai également avancé sur des recettes packer de création de template pour proxmox et des recettes terraform pour du déploiement proxmox (mais j'ai encore pas mal d'issue sur le sujet pour le release en public).
Idem une fois fait pour proxmox je voudrais porter la partie terraform pour ansible et azure (mais ça sera bien plus tard ça).

Et sinon je vais également avancer sur un installer en python pour vérifier les versions de vagrant, ansible,...qui sont présentes sur le poste avant install et s'assurer de l'espace libre etc... bref rendre l'installation un peu plus user friendly ;)

voilà pour la roadmap en gros, merci pour votre commentaire en tout cas. Je ferais sûrement un post twitter quand tout cela sera release en public, wait & see 👍

**Comment by iNoSec2** [comment]
C'est vraiment top! Du gros travail et un futur lab AD aux petits oignons.
Encore une fois merci pour tout

**Comment by belveruski** [comment]
Merci pour ce travail et pour les informations concernant le suivi du projet.

**Comment by IscarioteSXIII** [comment]
Passer certain serveurs en core pourrait egalement etre une bonne solution pour ceux qui sont limités en ressources. Perso, j'ai un Poweredge a la maison, avec un ESXi, mais pour deployer le lab sur Virtualbox, obligé de travailler sur mon pc perso, qui n'a que 16Go de RAM un un vieux i7-4710....et c'est vraiment limite. Je suis obligé d'installer chaque playbook un a un, par serveur , pour ne pas risquer d'erreurs suite à la latence ou probleme lors de reboot. J'espere réussir à pouvoir modifié ton travail pour déployer les VMs, car la conversion Virtualbox ---> ESXi est une horreur .

**Comment by Mayfly277** [comment]
yes faudrait que je trouve un esxi pour faire des tests et voir ce qu'il est possible de faire.
